gulpfile.js

```diff
@@ -156,6 +157,18 @@ gulp.task('lint', function() {
     .pipe(eslint.failAfterError());
 }); 
 
+gulp.task('compress', ['resources', 'html'], function() {
+	  return gulp.src('dist/**/*.js')
```

Comment:
I understand it minifies angular dependencies as well? Is there a way to use angular.min.js instead?